scene/2d/physics/animatable_body_2d.cpp

```diff
@@ -85,16 +83,23 @@ void AnimatableBody2D::_notification(int p_what) {
 			set_notify_local_transform(false);
 		} break;
 
-		case NOTIFICATION_LOCAL_TRANSFORM_CHANGED: {
+		case NOTIFICATION_TRANSFORM_CHANGED: {
+			if (!sync_to_physics) {
+				return;
+			}
+			if (Engine::get_singleton()->is_editor_hint()) {
```

Comment:
```suggestion
			if (!sync_to_physics || Engine::get_singleton()->is_editor_hint()) {
```
To not clutter the code

Reply:
Of course, I missed this one. Thanks.